packages/flutter/test/material/time_picker_theme_test.dart

```diff
@@ -385,7 +396,7 @@ TimePickerThemeData _timePickerTheme() {
     hourMinuteShape: const RoundedRectangleBorder(borderRadius: BorderRadius.all(Radius.circular(16.0))),
     dayPeriodShape: const RoundedRectangleBorder(borderRadius: BorderRadius.all(Radius.circular(16.0))),
     dayPeriodBorderSide: const BorderSide(color: Colors.blueAccent),
-    inputDecorationTheme: const InputDecorationTheme(
+    inputDecorationTheme: !includeInputDecoration ? null : const InputDecorationTheme(
```

Comment:
I think this is more readable as includeInputDecoration ? cons InputDecorationTheme(...) : null

Reply:
Done